### PR TITLE
[MIG] delivery_carrier_warehouse: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**
Standard migration, no new feature added

### CODE REVIEW

**Review by simahawk — CHANGES_REQUESTED** [review]
@phuctranerp don't we miss the original commits?

**Comment by hailangvn** [comment]
Good day @simahawk , the commits are in  the branch already because the branch 14.0 was based on 13.0:
https://github.com/OCA/wms/pull/167: https://github.com/OCA/wms/commits/14.0/delivery_carrier_warehouse
https://github.com/OCA/wms/pull/177: https://github.com/OCA/wms/commits/14.0/stock_picking_type_shipping_policy
https://github.com/OCA/wms/pull/181: https://github.com/OCA/wms/commits/14.0/stock_picking_type_shipping_policy_group_by
https://github.com/OCA/wms/pull/184: https://github.com/OCA/wms/commits/14.0/stock_available_to_promise_release
https://github.com/OCA/wms/pull/185: https://github.com/OCA/wms/commits/14.0/delivery_carrier_preference

File: delivery_carrier_warehouse/views/stock_warehouse.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
             <field name="partner_id" position="after">
                 <field
                     name="delivery_carrier_id"
-                    domain="['|', ('company_id', '=', False), ('company_id', '=', company_id.id)]"
+                    domain="['|', ('company_id', '=', False), ('company_id', '=', company_id)]"
```
Comment (simahawk) [inline delivery_carrier_warehouse/views/stock_warehouse.xml:11]:
```suggestion
                    domain="['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]"
```
no?

**Comment by simahawk** [comment]
> Good day @simahawk , the commits are in the branch already because the branch 14.0 was based on 13.0:
> #167: https://github.com/OCA/wms/commits/14.0/delivery_carrier_warehouse
> #177: https://github.com/OCA/wms/commits/14.0/stock_picking_type_shipping_policy
> #181: https://github.com/OCA/wms/commits/14.0/stock_picking_type_shipping_policy_group_by
> #184: https://github.com/OCA/wms/commits/14.0/stock_available_to_promise_release
> #185: https://github.com/OCA/wms/commits/14.0/delivery_carrier_preference

ouch, didn't notice we had them all. Sorry, I beg your pardon :innocent:

File: delivery_carrier_warehouse/views/stock_warehouse.xml (hunk 1 above)
Comment (hailangvn) [reply delivery_carrier_warehouse/views/stock_warehouse.xml:11]: I am afraid not. I can only find `company_id` at https://github.com/odoo/odoo/blob/14.0/addons/stock/models/stock_warehouse.py#L34 and could not find `company_ids` in any inherited `stock.warehouse` of odoo/addons.

**Review by sebalix — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**